FirebasePerformance/Sources/Instrumentation/FPRNetworkTrace.m

```diff
@@ -191,9 +200,9 @@ - (void)start {
     [self checkpointState:FPRNetworkTraceCheckpointStateInitiated];
 
     FPRSessionManager *sessionManager = [FPRSessionManager sharedInstance];
-    [self updateTraceWithCurrentSession];
+    [self updateTraceWithCurrentSession:[[sessionManager sessionDetails] copy]];
```

Comment:
Is it still possible that `sessionManager.sessionDetails` is concurrently modified at the time of reading so it may trigger a similar crash?

Reply:
I guess we can declare [FPRSessionManager.sessionDetails](https://github.com/firebase/firebase-ios-sdk/blob/b19fb5f2e7326e64d23feb1db2311468f1b4e06d/FirebasePerformance/Sources/AppActivity/FPRSessionManager%2BPrivate.h#L25) property as `atomic` to prevent concurrent access.

Reply:
That is a good suggestion. Added protection by making it an atomic property.